mlrun/feature_store/steps.py

```diff
@@ -376,3 +496,33 @@ def do(self, event):
         for func in self._tagging_funcs:
             func(event)
         return event
+
+
+class DropFeatures(StepToDict, MLRunStep):
+    def __init__(self, features: List[str], **kwargs):
+        """Drop all the features from feature list
+
+        :param features:    string list of the features names to drop
+
+        example::
+
+            feature_set = fs.FeatureSet("fs-new",
+                                            entities=[fs.Entity("id")],
+                                            description="feature set",
+                                            engine="pandas",
+                                            )
+            # Pre-processing grpah steps
+            feature_set.graph.to(DropFeatures(features=["age"]))
+            df_pandas = fs.ingest(feature_set, data)
+
+        """
+        super().__init__(**kwargs)
+        self.features = features
+
+    def _do_storey(self, event):
+        for feature in self.features:
+            del event[feature]
```

Comment:
Are you sure all features are always present? This will raise an error otherwise.